actions/sharedconfig/get.go

```diff
@@ -38,7 +38,7 @@ func (a *Action) oldGet(c *gin.Context) {
 // @Description	Get shared config
 // @Tags		Configurations
 // @Produce		json
-// @Success		200 {object} models.SharedConfig "Shared configuration"
+// @Success		200 {object} response.SharedConfig "Shared configuration"
```

Comment:
Please add in PR description that you also updated something on sharedconfig model.

Reply:
but I will add this information in the description

Reply:
done